common.yml

```diff
@@ -740,6 +742,17 @@ functions:
         set -e
         GOPATH=$PWD go get github.com/10gen/ftdc-utils/cmd/ftdc
 
+  "fetch gometalinter" :
```

Comment:
gometalinter repo says: "DEPRECATED: Use https://github.com/golangci/golangci-lint"  Maybe we should use the recommended tool instead.